cmake/modules/AddSwift.cmake

```diff
@@ -1882,6 +1886,16 @@ function(add_swift_target_library name)
        NOT ${name} STREQUAL swiftRemoteMirror)
       list(APPEND swiftlib_link_flags_all "-Wl,-z,defs")
     endif()
+    # Setting back linker flags which are not supported when making Android build on macOS cross-compile host.
+    if(SWIFTLIB_SHARED)
+      if(sdk IN_LIST SWIFT_APPLE_PLATFORMS)
+        list(APPEND swiftlib_link_flags_all "-dynamiclib -Wl,-headerpad_max_install_names")
```

Comment:
Doesn't this break if you build static?

Reply:
In file `cmake/modules/SwiftSharedCMakeConfig.cmake` we changed settings only for shared libraries (`CMAKE_SHARED_LIBRARY_CREATE_C_FLAGS`). This PR not altering standard CMake settings for static libs.